src/types.ts

```diff
@@ -3855,6 +3914,8 @@ export declare class Vec2 {
 	toString(): string
 }
 
+export evaluateBezier(pt1: Vec2, pt2: Vec2, pt3: Vec2, pt4: Vec2, t: number): Vec2
```

Comment:
Currently all functions are exported in KaboomCtx [here](https://github.com/replit/kaboom/blob/f27b2220d3977820a32e541e502127bb6dd4adc2/src/kaboom.ts#L6421) even the pure functions